client/templates/nav.jade

```diff
@@ -8,3 +8,5 @@ template(name="nav")
         a.birt(href="{{pathFor 'birt'}}") BIRT
       li(class="{{isActiveRoute 'eidr'}}")
         a.eidr(href="{{pathFor 'eidr'}}") EIDR
+      li(class="{{isActiveRoute 'performance'}}")
+        a.performance(href="{{pathFor 'performance'}}") PERFORMANCE
```

Comment:
I'm not running the application locally but I imagine 'Performance' does not need to be all caps (if so we could use css to transform it)?